Dockerfile-on-prem

```diff
@@ -0,0 +1,9 @@
+FROM harbor.devops.kubesphere.local:30280/library/java:openjdk-8-jre-alpine
```

Comment:
I'd recommend that providing some description about the purpose of this file here.

Reply:
Where does the base image `harbor.devops.kubesphere.local:30280/library/java:openjdk-8-jre-alpine` comes from?

Reply:
I think we should make a tutorial about devops hand's lab in offline mode.